### PR TITLE
[Bugfix]: Links having long URLs not editable in data objects

### LINKED ISSUE #10753: [Bug]: Links having long URLs not editable in data objects
### Expected behavior

Edit links as expected

### Actual behavior

As the buttons are positioned using js, they get cut off when used long URLs:

image

Replaced by a shorter URL:

image

### Steps to reproduce

Use a long URL in a data object link field

### PR DESCRIPTION
**Pull request by karlozrilic**
## Changes in this pull request  
Resolves #10753

### CODE REVIEW

File: bundles/AdminBundle/Resources/public/js/pimcore/object/tags/link.js [hunk 1]
```diff
@@ -90,6 +90,10 @@ pimcore.object.tags.link = Class.create(pimcore.object.tags.abstract, {
             text = this.data.path;
         }
 
+        if (text.length > 100) {
+            text = text.slice(0, 97)
+            text += '...'
```
Comment (brusch) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/tags/link.js]:
```suggestion
            text += '…'
```
Let's use ellipsis here 😉

File: bundles/AdminBundle/Resources/public/js/pimcore/object/tags/link.js [hunk 2]
```diff
@@ -165,6 +169,11 @@ pimcore.object.tags.link = Class.create(pimcore.object.tags.abstract, {
...
```
Comment (brusch) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/tags/link.js]:
```suggestion
            text += '…'
```

**Review by brusch — CHANGES_REQUESTED** [review]

File: bundles/AdminBundle/Resources/public/js/pimcore/object/tags/link.js (hunk 1 above)
Comment (NiklasBr) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/tags/link.js]: 👍 … and in that case it is also possible to use `text.slice(0, 99)`

**Comment by brusch** [comment]
Thanks! 🙏

**Pull request merged**